mona_sdk/client.py

```diff
@@ -632,7 +664,80 @@ def get_config_history(self, number_of_revisions=UNPROVIDED_VALUE):
             "get_config_history", data={"number_of_revisions": number_of_revisions}
         )
 
+    def _update_sampling_factors_if_needed(self):
+        """
+        If the client was initiated with a sampling config name, check if the
+        configuration was changed since the client vars were assigned, and if so, update
+        them accordingly.
+        """
+        if self._sampling_config_name:
```

Comment:
nit - instead of wrapping the whole function just switch the bool to "if not..." and make early return. It saves some un-needed indentation (and is a common practice)